octobot_trading/producers/ohlcv_updater.py

```diff
@@ -82,15 +85,26 @@ async def _initialize_candles(self, time_frame, pair):
         candles: list = await self.channel.exchange_manager.exchange \
             .get_symbol_prices(pair, time_frame, limit=self.OHLCV_OLD_LIMIT)
         self.channel.exchange_manager.uniformize_candles_if_necessary(candles)
-        if candles:
+        if len(candles) > 1:
+            self._set_init_attempts(pair, time_frame, 0)
             await self.channel.exchange_manager.get_symbol_data(pair) \
                 .handle_candles_update(time_frame, candles[:-1], replace_all=True, partial=False)
         else:
             # When candle history cannot be loaded, retry to load it later
-            self.logger.error(f"Failed to init candle history")
-            asyncio.get_event_loop().call_later(self.OHLCV_INITIALIZATION_RETRY_DELAY,
-                                                asyncio.create_task,
-                                                self._initialize_candles(time_frame, pair))
+            self.logger.warning(f"Failed to initialize candle history for {pair} on {time_frame}. Retrying in "
+                                f"{self.OHLCV_INITIALIZATION_RETRY_DELAY} seconds")
+            try:
+                self.candles_init_retry[pair][time_frame] += 1
```

Comment:
I think we don't care of symbol and tf, if it doesn't work for 1, it wont work for all. We should just limit global retry count in my opinion

Reply:
We can do that but in this case it's more complicated to manage since there is a request for each pair * time_frames, you think we should do that instead ?

Reply:
Do you think it'll be more complicated in this way ? I thought it would be less